### PR TITLE
cd, translation-templates/*: add Czech translation

### PR DESCRIPTION
**Pull request by AloisSeckar**
I would like to start helping with translation into my native language - Czech. From reading docs I didn't understand if starting a new language is possible just like that, or if something more has to be done. Please point me out to right direction, if I did something wrong.

### CODE REVIEW

File: contributing-guides/translation-templates/common-arguments.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 | bn    |                      |                        |                                   |           |                   |
 | bs    |                      |                        |                                   |           |                   |
 | ca    | camí/al/fitxer       | camí/al/directori      | camí/al/fitxer_o_directori        | paquet    | nom_usuari        |
+| cs    | cesta/k/souboru      | cesta/k/adresari       | cesta/k/souboru_ci_adresari       | balíček   | username          |
```
Comment (kbdharun) [inline contributing-guides/translation-templates/common-arguments.md:14]: Kinda curious, Isn't there an equivalent word for username in Czech? Or do you guys prefer using the English word?

_This is what I found on Google translate uživatelské jméno_

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, I have some suggestions/queries for this page.

Thanks for taking the initiative to add the Czech language to tldr (and also adding all the translation templates).

File: contributing-guides/translation-templates/common-arguments.md (hunk 1 above)
Comment (AloisSeckar) [reply contributing-guides/translation-templates/common-arguments.md:14]: This is my bad, sorry. I didn't noticed the line wraps up to next screen row, I thought "packet" is the last entry. Will update asap.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]
Remember to update `set-info-link.py` soon.

**Pull request merged**